### PR TITLE
New annotation import format-video

### PR DESCRIPTION
**Pull request by ovalle15**
Created examples with all the annotation types supported for videos Created all examples with new NDJSON format annotations

### CODE REVIEW

File: examples/annotation_import/video.ipynb [hunk 1]
```diff
@@ -1,346 +1,825 @@
 {
```
[the remark this answers is not in the document]
Comment (ovalle15) [reply examples/annotation_import/video.ipynb:1]: <p>mmm I couldn't get it to work. I can take a look tomorrow again.</p>

[the remark this answers is not in the document]
Comment (ovalle15) [reply examples/annotation_import/video.ipynb:1]: Done, a global nested classification works.

[the remark this answers is not in the document]
Comment (ovalle15) [reply examples/annotation_import/video.ipynb:1]: Please see https://github.com/Labelbox/labelbox-python/pull/806

File: examples/annotation_import/video.ipynb [hunk 2]
```diff
@@ -1,346 +1,1016 @@
...
```
Comment (msokoloff1) [inline examples/annotation_import/video.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    frame_bbox_with_checklist_subclass_ndjson = {
</pre><p>Can you just make a note here as to what we are doing. Just mention that we do not support classifications by name and that we will populate these later in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (ovalle15) [reply examples/annotation_import/video.ipynb:1]: <p>Done !</p>

**Review by msokoloff1 — APPROVED** [review]

**Pull request merged**